src/database/models.py

```diff
@@ -41,4 +42,28 @@ class Occurrence(Base):
         nullable=False)
 
 
+class Neighborhood(Base):
+    __tablename__ = 'neighborhood_stay_safe'
+
+    id_neighborhood = Column(Integer, primary_key=True)
+    neighborhood = Column(String, nullable=False)
+    city = Column(String, nullable=False)
+    state = Column(String, nullable=False)
+    rating = relationship("Rating")
+
+
```

Comment:
Put the maximum sizes in the Neighborhood model string fields, as described in the issue